backend/box/box.go

```diff
@@ -98,6 +128,93 @@ func init() {
 	})
 }
 
+func getBoxConfig(configFile string) ConfigJSON {
+	jsonData, err := os.Open(configFile)
```

Comment:
This open appears to be missing a Close?

You might be better off using ioutil.ReadFile rather than Open/ReadAll/Close